.github/workflows/test-move-prover.yml

```diff
@@ -1,38 +1,78 @@
 name: Test_Move_Prover
 
 on:
+  pull_request:
```

Comment:
you probably want to remove this, as this will get triggered for PRs against this repo

Reply:
OK.